errors/invalid-page-config.md

```diff
@@ -23,17 +24,50 @@ config.amp = true
 
 This is not allowed
 
+```js
+export const config = {
+  amp: 1 + 1 > 2,
+}
+```
+
+This is not allowed
+
```

Comment:
**suggestion:** it is hard to spot what is allowed and what is not. 

What about using a table, or bold, or any artifact helping readers to immediately find what is allowed and what is not?

<table>
<thead>
  <tr><th>Not allowed</th><th>Allowed</th></tr>
</thead>
<tbody>
  <tr><td>

```js
export const config = {
  amp: 1 + 1 > 2,
}
```
  </td><td>

```js
export const config = {
  amp: false,
}
```

  </td></tr>
  <tr><td>

   ```js
   export const config = {
     runtime: `n${'od'}ejs`
   }
   
   export const config = {
     runtime: (() => 'nodejs')()
   }
   ```

   </td><td>

   ```js
   export const config = {
     runtime: `nodejs`,
   }
   
   export const config = {
     runtime: 'nodejs',
   }
   ```

   </td></tr>
</tbody>
</table>

Reply:
@feugy Agreed.

An alternative I have thought of is to replicate the ESLint documentation style:
https://eslint.org/docs/latest/rules/valid-typeof

Reply:
Update in [`1858c1a` (#38750)](https://github.com/vercel/next.js/pull/38750/commits/1858c1a2fa0359bb58c8abf3cdb7ee93796aa6b7)